core/scheduler/metrics.go

```diff
@@ -56,7 +59,14 @@ var (
 		Namespace: "core",
 		Subsystem: "scheduler",
 		Name:      "validator_balance_gwei",
-		Help:      "Total balance of a validator by public key by status",
+		Help:      "Total balance of a validator by public key",
+	}, []string{"pubkey_full", "pubkey"})
+
+	statusGauge = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Namespace: "core",
+		Subsystem: "scheduler",
+		Name:      "validator_status",
+		Help:      "Constant gauge with label set to current status of the validator by pubkey",
```

Comment:
Gauge with validator pubkey and status as labels, value=1 is current status, value=0 is previous.

Reply:
not really "constant"...